tests/import_specifications/test_file_writers.py

```diff
@@ -1,17 +1,27 @@
+import os
 import uuid
 
+import openpyxl
+
 from collections.abc import Generator
 from pathlib import Path
 from pytest import raises, fixture
+from typing import Any
```

Comment:
I missed that this was unused after I moved the Excel checker to the test utils file. I've removed it locally, it'll be fixed in a future PR.